mps_youtube/player.py

```diff
@@ -374,14 +375,24 @@ def _get_input_file():
 def _launch_player(song, songdata, cmd):
     """ Launch player application. """
 
+    #song.title
+
     util.dbg("playing %s", song.title)
     util.dbg("calling %s", " ".join(cmd))
 
     # Fix UnicodeEncodeError when title has characters
     # not supported by encoding
     cmd = [util.xenc(i) for i in cmd]
 
-    arturl = "https://i.ytimg.com/vi/%s/default.jpg" % song.ytid
+    metadata = _get_metadata(song.title)
+
+    if metadata == None :
+        arturl = "https://i.ytimg.com/vi/%s/default.jpg" % song.ytid
+        metadata = (song.ytid, song.title, song.length, arturl, [''], '')
+    else :
+        arturl = metadata['album_art_url']
+        temp = (song.ytid, metadata['track_title'], song.length, arturl, [metadata['artist']], metadata['album'])
+        metadata = temp
```

Comment:
we can assign to metadata directly, right?

Reply:
I don't know why I did that. XD